Update coding-style.md

hareshambaliya opened the pull request:
Changed Keyboard shortcut to show white space. There was a wrong shortcut of View White Space (Ctrl+E, S) and correct one is (Ctrl+R, Ctrl+W). I don't know is it allows us to contribute/change in guidelines or not. It's my first PR. I hope it will be accepted.

hareshambaliya committed:
Update coding-style.md

Changed Keyboard shortcut to show white space.

gfoidl commented on Documentation/coding-guidelines/coding-style.md:
```diff
@@ -20,7 +20,7 @@ The general rule we follow is "use Visual Studio defaults".
 8. Avoid spurious free spaces.
    For example avoid `if (someVar == 0)...`, where the dots mark the spurious free spaces.
-   Consider enabling "View White Space (Ctrl+E, S)" if using Visual Studio to aid detection.
+   Consider enabling "View White Space (Ctrl+R, Ctrl+W)" if using Visual Studio to aid detection.
```
Maybe add "in the default configuration".

If someone has different shortcuts configured, it won't match again :wink:

safern reviewed (approved):
Looks good, thanks @hareshambaliya -- could you please address @gfoidl's comment? As people could have different configuration?

hareshambaliya committed:
Update coding-style.md

@gfoidl I have added about how to enable View White Space. I think we are good now.
